src/NUnitEngine/nunit.engine.core/ITestAgent.cs

```diff
@@ -24,18 +24,53 @@
 namespace NUnit.Engine
 {
     /// <summary>
-    /// The ITestAgent interface is implemented by remote test agents.
+    /// Defines the current communication protocol between the engine and the agent. This is an implementation detail
+    /// which is in the process of changing as the dependency on .NET Framework's remoting is removed.
     /// </summary>
     public interface ITestAgent
     {
         /// <summary>
         /// Stops the agent, releasing any resources
         /// </summary>
-        void Stop();
+        void ShutDown();
```

Comment:
Others may not be confused, but I guess I find it confusing to see an interface that implements `ShutDown` even though it's not an `IService` and which implements it without also implementing `StartUp`.